---
layout: post
title: RNA Extraction and cDNA Library Prep for Pentagona and Hystera Sea Stars
tags: [ RNA, cDNA, Pentagona, Hystera, Library Prep ]
---

# Extraction RNA and Stranded mRNA Library Prep for Pentagona and Hystera Sea Stars for Use in Genome Building

### RNA Extraction

Used [Zymo Research Quick DNA/RNA Miniprep Plus Kit](https://www.zymoresearch.com/collections/quick-dna-rna-kits) for extraction (probably should just have used the RNA one) and the [Zymo bashing beads](https://www.zymoresearch.com/products/zr-bashingbead-lysis-tubes-0-1-0-5-mm) for homogenization. Roughly followed [Emma Strand's Soft Homogenization Protocol](https://github.com/emmastrand/EmmaStrand_Notebook/blob/master/_posts/2019-06-05-Soft-and-Hard-Homogenization-Protocol.md) for ideas on how to homogenize samples because I'd never worked with sea stars before and Emma consistently gets good RNA with her protocol. Additionally I looked at [this version of Mo'orea coral extractions](https://meschedl.github.io/MESPutnam_Open_Lab_Notebook/Eggs-Bundles-Moorea/) which was the only time when I got good RNA out of those samples, and they are also tissue preserved in DNA/RNA shield, so that is why I had no incubation step.

1. Took out 1 P6 and 1 H6 sample (tissue in DNA/RNA shield) and thawed on ice
2. Filled two ZR bashing bead tubes with 700μl DNA/RNA Shield and named the tubes PA and HA
3. Sterilized foil, scalpel, foil, and forceps. Took sample out of tube and cut in half on foil. Placed one half in the new bashing bead tube
4. Vortexed each sample for 1 minute max speed
5. Let sit after votexing: very bubbly. _HA liquid was a lot more orange than PA. There was still a pretty prominent ossicle chunk left in there_
6. Aspirated all liquid I could get off into new labeled 1.5mL tubes
7. Took 300μl of that into new 1.5mL tubes going forward
8. Added 30μl prok digestion buffer
9. Added 15μl Pro K
10. Vortexed and spun down tubes


11. Equal volumes of DNA/RNA Lysis Buffer were added to each sample tube: 345μl
12. Mixed samples by flicking and spinning down
13. ~700µl of sample was gently added to Yellow DNA spin columns
14. Centrifuged columns at 16000 rcf for 30 seconds
15. Flow-through was transferred to new 1.5 mL tubes for RNA
8. Added 400µl DNA/RNA Prep Buffer gently to the yellow DNA spin columns
9. Centrifuged at 16,000 rcf (g) for 30 seconds
10. Discarded flow through (Zymo kit waste)
11. Added 700µl DNA/RNA Wash Buffer gently to the yellow DNA spin columns
12. Centrifuged at 16,000 rcf (g) for 30 seconds
13. Discarded flow through (Zymo kit waste)
14. Added 400µl DNA/RNA Wash Buffer genetly to the yellow DNA spin columns
15. Centrifuged at 16,000 rcf (g) for **2 minutes**
16. Discarded flow through (Zymo kit waste)
17. Transferred yellow columns to new 1.5mL microcentrifuge tubes
18. Add 50µl warmed 10mM Tris HCl to each yellow DNA column
19. Incubated at room temp for 5 minutes
20. Centrifuged at 16,000 rcf (g) for 30 seconds
21. Repeated last three steps for a final elution volume of 100µl


30. Added equal volume  100% EtOH to the 1.5mL tubes labeled for RNA containing the original yellow column flow through: ~700µl
31. Vortexed and spin down to mix
25. Added 700µl of that liquid to the green RNA spin columns
26. Centrifuged at 16,000 rcf (g) for 30 seconds
27. Discarded flow through (Zymo kit waste)
28. Added 700µl to the green RNA spin columns (the rest from the 1.5mL RNA tubes)
29. Centrifuged at 16,000 rcf (g) for 30 seconds
30. Discarded flow through (Zymo kit waste)
31. Added 400µl DNA/RNA Wash Buffer gently to each green RNA column
32. Centrifuged at 16,000 rcf (g) for 30 seconds
33. Discarded flow through (Zymo kit waste)
34. Made DNase I treatment master mix:
    - 75µl DNA Digestion buffer x 2 = 150µl
    - 5µl DNase I x 2 = 10µl
35. Added 80µl DNase I treatment master mix directly to the filter of the green RNA columns
36. Incubated at room temp for 15 minutes
37. Added 400µl DNA/RNA Prep Buffer gently to each column
38. Centrifuged at 16,000 rcf (g) for 30 seconds
39. Discarded flow through (Zymo kit waste)
40. Added 700µl DNA/RNA Wash Buffer gently to the yellow DNA spin columns
41. Centrifuged at 16,000 rcf (g) for 30 seconds
42. Discarded flow through (Zymo kit waste)
43. Added 400µl DNA/RNA Wash Buffer genetly to the yellow DNA spin columns
44. Centrifuged at 16,000 rcf (g) for **2 minutes**
45. Discarded flow through (Zymo kit waste)
46. Transferred green columns to new 1.5mL microcentrifuge tubes
47. Added 50µl warmed DNase/RNase free water to each green RNA column by dripping slowly directly on the filer
48. Incubated at room temp for 5 minutes
49. Centrifuged at 16,000 rcf (g) for 30 seconds
50. ed last three steps for a final elution volume of 100µl
51. Labeled 1.5mL tubes on ice afterwards, and aliquoted 5µl into PCR strip tubes to save for Qubit and Tape Station to avoid freeze-thaw
52. Stored all tubes in the -80

### Qubit and TapeStation

|Sample|DNA Standard 1 (RFU)|DNA Standard 2 (RFU)|DNA 1 (ng/µl)|DNA 2 (ng/µl)|Average DNA| RNA Standard 1 (RFU)| RNA Standard 2 (RFU)| RNA 1 (ng/µl)|RNA 2 (ng/ul)|Average RNA|
|------|----------|----------|-------------|-------------|-------------|-------------|----|----|----|----|
|PA|181|20099|66|65.4|**65.8**|385|10633|18.8|18.4|**18.6**|
|HA|181|20099|16.9|16.7|**16.8**|385|10633|19.6|19.2|**19.4**|

Full TapeStation[Results](https://drive.google.com/open?id=1rk8AQl_Q6bFs29cQqiXdH9EgIjcp6kfI):

![1]({{ site.baseurl}}/images/PA-RNA.png "1")
![2]({{ site.baseurl}}/images/HA-RNA.png "2")


### Stranded mRNA Seq Prep

Samples are concentrated enough for the library prep, so they were vacufuged for ~2 hours and then re-qubited. PA: 42.8ng/µl. HA: 47.4ng/µl.

**Capture**

1. Took out mRNA capture beads, mRNA bead binding buffer, and mRNA bead wash buffer from 4 degree and let get to room temp
2. Resuspended mRNA capture beads by pipetting up and down
3. Calculated the number of beads needed:  
    `26.25μl beads * 2 samples = 52.5μl`
4. Pipetted 52.5μl mRNA capture beads into a new PCR tube and put on the plate magnet stand
5. Removed supernatant
6. Added 52.5μl of bead binding buffer and pipetted to mix off magnet
7. Put back on magnet and removed as much supernatant as possible
8. Added 52.5μl of bead binding buffer again and pipetted to mix off magnet
9. Put back on magnet and again removed as much supernatant as possible
10. Added 52.5μl of bead binding buffer and pipetted to mix off magnet
11. Added 1μg of RNA from each sample into new PCR tubes and filled the rest up to 25μl with nuclease free water:
  - PA: 23.36μl RNA and 1.64μl nuclease-free water
  - HA: 21.28μl RNA and 3.72μl nuclase-free water
11. Added 25μl of the resuspended mRNA capture beads to each sample tube
12. Placed tubes in the 1st mRNA capture program in the thermocycler
13. Placed tubes on the magnet plate and removed all supernatant when the solution went clear
14. Removed tubes from the magnet plate and resuspended beads in 100μl of bead wash buffer
15. Placed tubes on the magnet plate and removed all of the supernatant when the solution went clear
16. Resuspended beads in 25μl of RNase-free water off magnet
17. Placed tubes in the 2nd mRNA capture program in the thermocycler
18. Took tubes out of the thermocycler and added 25μl of bead binding buffer to each tube and pipetted to mix
19. Incubated the tubes on the shaker at 200rpm for 5 minutes
20. Made 1X Fragment, Prime, and Elute Buffer on ice bucket:  
  `5.5μl water * 2.2 = 12.1μl`     
  `5.5μl 2X FPE buffer * 2.2 = 12.1μl `
21. Placed tubes on the magnet plate and removed supernatant when the solution went clear
22. Resuspended beads off magnet in 11μl 1X FPE buffer
23. Placed tubes in the 4 degree fridge overnight (safe stopping point)

**Fragmentation**

23. Took tubes out of fridge and put in thermocycler for RNA fragmentation program 300-400bp (6 minutes at 85 degrees)
24. Made 1st Strand Synthesis Master Mix on ice:  
    `5.5μl 1st strand synthesis buffer * 2.2 = 12.1μl`  
    `.5μl KAPA script * 2.2 = 1.1μl`
25. IMMEDIATELY placed tubes on magnet plate once the program was finished (some tubes had popped open)
26. Removed 10μl of clear supernatant and placed in new PCR strip tubes on ice

**1st Strand Synthesis**

27. Added 5μl of the 1st strand synthesis master mix to each tube and pipetted to mix
28. Placed in thermocycler 1st strand synthesis program
29. Made 2nd Strand Synthesis and Marking Master Mix on ice:  
    `15.5μl 2nd strand marking buffer * 2.2 = 34.1μl`  
    `1μl second strand enzyme * 2.2 = 2.2μl`
30. Removed tubes from the thermocycler and placed on ice

**2nd Strand Synthesis**

31. Added 15μl of the 2nd strand synthesis and marking master mix and pipetted to mix
32. Put in thermocycler 2nd strand synthesis program
33. Took KAPA Pure Beads out of the 4 degree and swirled them to mix and let get to room temp
34. Took tubes out of the thermocycler and added 54μl KAPA Pure Beads, pipetting to mix
35. Incubated tubes on shaker for 15 minutes
36. Made A-tailing Immediately Master Mix on ice:  
    `12μl water * 2.2 = 26.4μl`  
    `1.5μl 10X A-tailing buffer * 2.2 = 3.3μl`
    `1.5μl A-Tailing enzyme * 2.2 = 3.3μl`
37. Made fresh 80% EtOH for that day
38. Placed tubes on magnet plate and removed 80μl clear supernatant
39. Added 200μl 80% EtOH to each tube
40. Removed EtOH from each tube
41. Added 200μl 80% EtOH to each tube
42. Removed ALL EtOH from each tube, using a p20 to get extras and droplets of EtOH

**A-Tailing**

43. Waited ~30 seconds or less and resuspended the beads in 15μl of the A-tailing Immediately Master Mix
44. Put tubes in the thermocycler A-tailing program

**Adapter Ligation**

45. Made the Adapter Ligation Master Mix:  
  `8μl nuclease-free water * 2.2 = 17.4μl`  
  `7μl ligation buffer * 2.2 = 15.4μl`  
  `2.5μl DNA ligase * 2.2 = 5.5μl`  
5. Added 17.5μl of the ligation master mix to each tube out of the theromocylcer
6. Added 2.5μl of the diluted/annealed/working stock 700nM No-Barcode (NOBO) adapters to each sample
7. Pipetted to mix
8. Placed tubes on shaker for ~1hour room temp

**2 Cleanups**

50. Added 35μl of room temperature PEG to each sample and pipetted to mix
51. Placed on shaker for 15 minutes
12. Made fresh 80% EtOH
13. Placed tubes on magnet plate
14. Removed 67μl of supernatant from each tube
15. Added 100μl 80% EtOH to each tube  
16. Removed 100μl of supernatant from each tube
17.  Added 100μl 80% EtOH to each tube  
18. Removed ALL of the supernatant from each tube, using a p20 pipette tip to get rid of droplets
19. Resuspended beads in 25μl of 10mM Tris HCl pH 8
20. Placed tubes on shaker for 5 minutes
21. Added 25μl of room temperature PEG to each tube and pipetted to mix
22. Placed on shaker for 15 minutes
23. Made fresh 80% EtOH
24. Placed tubes on magnet plate
25. Removed 45μl of supernatant from each tube
26. Added 100μl 80% EtOH to each tube  
27. Removed 100μl of supernatant from each tube
28. Added 100μl 80% EtOH to each tube  
29. Removed ALL of the supernatant from each tube, using a p20 pipette tip to get rid of droplets
30. Resuspended beads in 11μl of 10mM Tris HCl pH 8
31. Placed tubes on shaker for 5 minutes
32. Placed tubes on magnet stand and removed 10μl of clear supernatant to new tubes

**Library Amplification**

To increase diversity of index sequences in sequencing (more color variation in the machine) the libraries were split at this point to get two different index combos per sample: PAmp1, PAmp2, HAmp1, and HAmp2. This means only 5μl of the above sample went in to each PCR, so the extra volume to 25μl was made up with water and the primer volume added was halved to keep the ratio the same.

73. Made Library Amplification Master Mix:  
  `12.5μl KAPA HotStart Ready Mix x 4.2 = 52.5μl`  
  `6μl nuclease-free water x 4.2 = 25.2μl`
74. Made PCR reaction tubes with these reagents/samples:

|Sample|Amount of cDNA|Library MM|index 1| index 2|
|---|----|----|----|----|
|PAmp1|5μl PA|18.5μl|.75μl 505|.75μl 709|
|PAmp2|5μl PA|18.5μl|.75μl 502|.75μl 710|
|HAmp1|5μl HA|18.5μl|.75μl 503|.75μl 705|
|HAmp2|5μl HA|18.5μl|.75μl 511|.75μl 703|

75. Pipetted to mix and placed in thermocyler 12 cycle PCR program (normal for this prep)
76. Took out KAPA beads
77. Did a 1X bead cleanup: 25μl beads to each sample
78. Performed normal bead clean up (see above for examples)
79. Resuspended and eluted beads in 22μl 10mM Tris HCl

**Broad Range Qubit and TapeStation**

PAmp1: 111ng/μl
PAmp2: 102ng/μl
HAmp1: 103ng/μl
HAmp2: 101ng/μl

Full TapeStation [Report](https://drive.google.com/open?id=1BRfSJgUlF-SnH-yDu3p5JoEFCMhAkLLk)

![3]({{ site.baseurl}}/images/PAmp1.png "1")
![4]({{ site.baseurl}}/images/PAmp2.png "2")
![5]({{ site.baseurl}}/images/HAmp1.png "1")
![6]({{ site.baseurl}}/images/HAmp2.png "2")

These worked well but they are smaller than we expected/wanted. Looks like they fragmented the RNA to ~200bp not 300bp.... Still sent to Seq tho!
